### PR TITLE
added single product screen

### PR DESCRIPTION
**Pull request by mrayor**
- Added Single Product Screen

### CODE REVIEW

File: src/components/home/components/CarouselCard.jsx [hunk 1]
```diff
@@ -33,20 +32,21 @@ export const CarouselCard = ({ item }) => {
...
+        <Text style={priceText}>
+          {NGN}
+          {price}
+        </Text>
       </View>
       <View style={iconsWrapper}>
-        {fontIcons.map(({ name }, index) => (
-          <FontAwesome key={index} name={name} size={20} />
-        ))}
+        <ContactsIcons size={20} icons={fontIcons} />
```
Comment (ekediala) [inline src/components/home/components/CarouselCard.jsx]: Why the hardcoded value?

File: src/components/home/components/Review.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+const { REVIEWS } = SCREEN_TEXT;
+const { businessNameText, reviewsWrapper, reviewText } = CarouselCardStyles;
+
+const Review = ({ businessName, ratings, review }) => {
+  return (
+    <View>
+      <Text style={businessNameText}>{businessName}</Text>
+      <View style={reviewsWrapper}>
+        <StarRatings ratings={ratings} size={13} />
```
Comment (ekediala) [inline src/components/home/components/Review.jsx]: Same here. Hardcoded size value

File: src/components/screens/navigators/Home.jsx [hunk 3]
```diff
@@ -1,31 +1,64 @@
...
     Filter: {
-      screen: FilterScreen
+      screen: FilterScreen,
+      navigationOptions: ({ navigation }) => {
+        return {
+          headerTitle: FILTER_SCREEN,
+          headerLeft: (
+            <AntDesign
+              name={CLOSE}
+              size={25}
```
Comment (ekediala) [inline src/components/screens/navigators/Home.jsx]: Again here

File: src/components/singleProduct/components/ProductDescription.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <View style={container}>
+      <Text style={textStyles}>
+        {CONDITION_TEXT}
+        {condition}
+      </Text>
+      <Divider style={seperator} />
+      <Text style={textStyles}>{description}</Text>
+      <View style={contactsWrapper}>
+        <ContactsIcons size={20} icons={fontIcons} />
```
Comment (ekediala) [inline src/components/singleProduct/components/ProductDescription.jsx]: Guess there's a reason you keep doing this. I do not know what the reason is so I'm just going to ask why.

Comment (mrayor) [reply src/components/singleProduct/components/ProductDescription.jsx]: I don't fully get what you mean though. That is the size value for the font icons. You want the size to be dynamically determined?

**Pull request merged**